doc/reverse_engineering/game_mechanics/market.md

```diff
@@ -74,3 +76,7 @@ If you happen to have more than one market, trade carts will choose the market w
 
 * You can trade with enemy markets.
 * Full trade carts cannot be loaded onto a transport ship. As soon as you give the order to board the ship, they change into an empty cart, dropping all of their gold.
+
+--------
+
+[here](https://gist.github.com/Piruzzolo/a0a66c3b592b3cadff785649ae8d7410) the (partly) decompiled CalculateTradeGold function
```

Comment:
Although this is nice, I would advise against linking decompiled source because it could get us in legal trouble. In general we should avoid decompiling and stick to deriving functions from observation or use secondary sources such as the AoE2 Wiki. This will be more work, but it usually keeps lawyers away.